pnpm-workspace.yaml

```diff
@@ -7,6 +7,12 @@ onlyBuiltDependencies:
 
 strictDepBuilds: true
 
+autoInstallPeers: false
```

Comment:
Because we should be aware before breaking Yarn folks/Eli's admitting niche pnpm setup.